tuning/punet_autotune.py

```diff
@@ -41,36 +41,70 @@ def get_dispatch_compile_command(
         return command
 
     def get_dispatch_benchmark_command(
-        self, candidate_tracker: libtuner.CandidateTracker
+        self,
+        candidate_tracker: libtuner.CandidateTracker,
     ) -> list[str]:
         compiled_vmfb_path = candidate_tracker.compiled_dispatch_path
         assert compiled_vmfb_path is not None
+
         command = [
-            "./benchmark_dispatch.sh",
-            compiled_vmfb_path.as_posix(),
+            "timeout",
+            "16s",
+            "./tools/iree-benchmark-module",
+            f"--device={libtuner.DEVICE_ID_PLACEHOLDER}",
+            f"--module={compiled_vmfb_path.resolve()}",
+            "--hip_use_streams=true",
+            "--hip_allow_inline_execution=true",
+            "--batch_size=1000",
+            "--benchmark_repetitions=3",
         ]
+
         return command
 
     def get_model_compile_command(
         self, candidate_tracker: libtuner.CandidateTracker
     ) -> list[str]:
         mlir_spec_path = candidate_tracker.spec_path
         assert mlir_spec_path is not None
+        target_dir = mlir_spec_path.resolve().parent.parent.parent
+        output_name = f"unet_candidate_{candidate_tracker.candidate_id}.vmfb"
         command = [
-            "./compile_unet_candidate.sh",
-            "winograd",
-            mlir_spec_path.as_posix(),
+            "timeout",
+            "300s",
+            "../int8-model/compile-punet-base.sh",
```

Comment:
Instead of `..`, can you query the script directory and use `str(script_dir / "../int8-model/...")`?